api/v1alpha4/awscluster_conversion.go

```diff
@@ -52,6 +52,7 @@ func (src *AWSCluster) ConvertTo(dstRaw conversion.Hub) error {
 // Assumes restored and dst are non-nil.
 func restoreControlPlaneLoadBalancer(restored, dst *infrav1.AWSLoadBalancerSpec) {
 	dst.Name = restored.Name
+	dst.ClassicELBProtocol = restored.ClassicELBProtocol
```

Comment:
Just confirming: not adding this failed our fuzzy conversion tests, right?
There was an issue about them not catching errors.

Reply:
Yes it's creating problem in fuzzy conversion tests that's why added this is v1alpha3 and v1alpha4 conversions.

Reply:
is it related to this [issue](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3085)

Reply:
No, it was about EKS conversion tests. Just wanted to confirm for myself, not related to this issue.